lib/telephony/pinpoint/opt_out_manager.rb

```diff
@@ -0,0 +1,48 @@
+module Telephony
+  module Pinpoint
+    class OptOutManager
+      # @return [Response]
+      def opt_in_phone_number(phone_number)
+        if Telephony.config.pinpoint.sms_configs.empty?
+          return PinpointHelper.handle_config_failure(:sns)
+        end
+
+        response = nil
+        Telephony.config.pinpoint.sms_configs.each do |config|
+          client = build_client(config)
+          next if client.nil?
+
+          opt_in_response = client.opt_in_phone_number(phone_number: phone_number)
+
+          return Response.new(success: opt_in_response.successful?)
```

Comment:
Is something here supposed to be assigning to the `response` variable above? Otherwise it seems unused and would always trigger the fallback failure?

Reply:
I think I should remove that variable. this `return` returns from the method because Ruby is weird about `return` inside of blocks

Reply:
26c828d1343b51b28ff689efd9c0e52b2cf8c33f